thirdeye/thirdeye-frontend/app/pods/components/composite-anomalies/component.js

```diff
@@ -78,9 +119,19 @@ export default Component.extend({
      * @param {Number} index
      *   The index of the breadcrumb that was clicked
      */
-    onBreadcrumbClick({ isRoot = false }, index) {
+    onBreadcrumbClick({ id, isRoot = false }, index) {
       this.set('breadcrumbList', this.breadcrumbList.splice(0, index + 1));
 
+      if (isRoot) {
+        const { output } = parseRoot(id, this.anomalies);
+
+        this.set('data', output);
+      } else {
+        const { output } = parseSubtree(id, this.anomalies);
+
+        this.set('data', output);
+      }
+
```

Comment:
This is mostly a stylistic thing so feel free to ignore, but would it make sense to use a ternary?   

``` 
const { output } = isRoot ? parseRoot(id, this.anomalies) : parseSubtree(id, this.anomalies);
this.set('data', output);
```
since the only difference is `parseRoot` vs `parseSubtree`

Reply:
Done.